cypress/integration/text.spec.ts

```diff
@@ -355,11 +359,53 @@ describe('Choices - text element', () => {
       });
     });
 
+    describe('allow html', () => {
```

Comment:
These look good. Can you add these tests to the `select-multiple` and `select-one` specs as well?

Reply:
Resolved in [#3633c4a](https://github.com/Choices-js/Choices/pull/984/commits/3633c4ac0f0a8437ea4ffdc44de0b872c8e816c5) and [#c989be1](https://github.com/Choices-js/Choices/pull/984/commits/c989be14917d58c6f0969218289299d42bf69338)